### PR TITLE
Increase spacing for suggestions in diagnostics

### PR DESCRIPTION
**Pull request by estebank**
Make the spacing between the code snippet and verbose structured
suggestions consistent with note and help messages.

r? @Centril

### CODE REVIEW

File: src/librustc_errors/emitter.rs [hunk 1]
```diff
@@ -1629,6 +1629,19 @@ impl EmitterWriter {
                             _ => ()
                         }
                     }
+                    if children.is_empty() && suggestions.iter().any(|s| {
+                        s.style != SuggestionStyle::CompletelyHidden
+                    }) {
+                        let mut buffer = StyledBuffer::new();
```
Comment (Centril) [inline src/librustc_errors/emitter.rs]: This logic seems identical (copied?) from above; can you de-dup?

File: src/librustc_errors/emitter.rs [hunk 2]
```diff
@@ -1629,6 +1629,19 @@ impl EmitterWriter {
...
+                        if !self.short_message {
+                            draw_col_separator_no_space(&mut buffer, 0, max_line_num_len + 1);
+                        }
+                        match emit_to_destination(&buffer.render(), level, &mut self.dst,
+                                                self.short_message) {
+                            Ok(()) => (),
+                            Err(e) => panic!("failed to emit error: {}", e)
+                        }
```
Comment (Centril) [inline src/librustc_errors/emitter.rs]: ...until here.

File: src/librustc_errors/emitter.rs (hunk 1 above)
Comment (estebank) [reply src/librustc_errors/emitter.rs]: done

**Comment by estebank** [comment]
Pinging because this has high potential for bitrot due to extensive touching of stderr files.

**Comment by JohnTitor** [comment]
It's a network error.
@bors retry

**Pull request merged**